shpc/main/modules/base.py

```diff
@@ -235,10 +240,9 @@ def docgen(self, module_name, registry=None, out=None, branch="main"):
         aliases = config.get_aliases()
         template = self.template.load("docs.md")
         registry = registry or defaults.github_url
-        github_url = "%s/blob/%s/%s/container.yaml" % (registry, branch, module_name)
-        raw_github_url = shpc.main.registry.get_module_config_url(
-            registry, module_name, branch
-        )
+        git_registry = self.registry.get_registry(registry, tag=branch)
+        github_url = git_registry.get_container_yaml_url(module_name)
+        raw_github_url = git_registry.get_raw_container_yaml_url(module_name)
```

Comment:
Could we tweak the naming a bit, do something like:

```python
remote = self.registry.get_registry(registry, tag=branch)
github_url = remote.get_container_url(module_name)
raw_github_url = remote.get_container_url(module_name, raw=True)
```

I'm suggesting removing yaml because it's the only file we have for a container (so sort of implied?)

Reply:
Agreed